hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1825,6 +1825,17 @@ func reconcileHostedControlPlane(hcp *hyperv1.HostedControlPlane, hcluster *hype
 		}
 	}
 
+	if hcp.Labels == nil {
+		hcp.Labels = make(map[string]string)
+	}
+	// All labels on the HostedCluster with this special prefix are copied
+	// Thoses are labels set by OCM
+	for key, val := range hcluster.Labels {
+		if strings.HasPrefix(key, "api.openshift.com") {
+			hcp.Annotations[key] = val
```

Comment:
Why are they set as annotations and not labels? and if annotations are intended, then lines 1828-1830 don't make sense.

Reply:
ohhh, right those should be labels. thanks